bindings/dotnet/build.py

```diff
@@ -25,54 +25,54 @@
 
 
 def classifier_to_target(classifier: str) -> str:
-    if classifier == 'osx-aarch_64':
-        return 'aarch64-apple-darwin'
-    if classifier == 'osx-x86_64':
-        return 'x86_64-apple-darwin'
-    if classifier == 'linux-x86_64':
-        return 'x86_64-unknown-linux-gnu'
-    if classifier == 'windows-x86_64':
-        return 'x86_64-pc-windows-msvc'
-    raise Exception(f'Unsupported classifier: {classifier}')
+    if classifier == "osx-aarch_64":
```

Comment:
Let's prevent this style changes unless we have a consistent rule. Otherwise changeset is forth and back.

Reply:
i.e., revert style changes.

Reply:
> Let's prevent this style changes unless we have a consistent rule. Otherwise changeset is forth and back.

Formatted by `ruff` which may later added in CI: https://github.com/apache/incubator-opendal/issues/3468

Reply:
anyway, reverted.